examples/kafka/confluent-s3-connector/kafka-blueprint.yaml

```diff
@@ -139,7 +139,7 @@ actions:
           containers:
           - name: container
             imagePullPolicy: Always
-        image: akankshakumari393/confluent-kafka-s3source:latest
+        image: akankshakumari393/confluent-kafka-s3source:v9.99.9-dev
```

Comment:
have we been using these images from your personal repo?

Reply:
we are not using confluent connectors anymore. For Kafka we are using the adobe sink and source connector.[https://github.com/kanisterio/kanister/tree/master/examples/kafka/adobe-s3-connector](https://github.com/kanisterio/kanister/tree/master/examples/kafka/adobe-s3-connector)